src/calendar/CSEvents.js

```diff
@@ -88,14 +86,12 @@ export default class CSEvents extends Base {
 	filterModel_Site = function (model) {
 		let siteFilter = this.filterValue('site')
 		// no filter
-		if (siteFilter == null) return true
-		// all sites event
-		if (model._original.site == null) return true
-		// return on id or name for legacy support
-		return (
-			siteFilter.includes('' + model._original.site.id) ||
-			siteFilter.includes('' + model._original.site.name)
-		)
+		if (siteFilter === null) return true
+
+		// all sites groups
```

Comment:
The comment here isn't immediately obvious to me why it has changed from "all sites event" - is this a copypasta from CSGroups?

Reply:
Just fixed this and added a multi site events test for the future functionality.